db/migrate/20210415015227_create_fulfillment.rb

```diff
@@ -0,0 +1,20 @@
+class CreateFulfillment < ActiveRecord::Migration[6.0]
+  def change
+    create_table :fulfillments do |t|
+      t.boolean :fulfilled, null: false, default: false
```

Comment:
So this was nullable before, but the default takes care of it now... yeah, that makes sense!

Reply:
exactly right! Nice and minor, but a quality of life improvement.